File: csharp/DVSClient/DVSClient/address/layout/LayoutLineVariable.cs
using DVSClient.Address.Layout.Elements;
using DVSClient.Server.Address.Layout;

namespace DVSClient.Address.Layout
{
    public class LayoutLineVariable : ILayoutLine
    {
        public string Name { get; set; }
        public int? MaxWidth { get; set; }

        public LayoutLineVariable(string name)
        {
            Name = name;
        }

        public LayoutLineVariable(RestApiAddressLayoutLine line)
        {
            Name = line.LineName ?? string.Empty;
            MaxWidth = line.MaxWidth;
        }

        public IEnumerable<IAddressElement?> GetElements()
        {
            return new List<IAddressElement?>();
        }
    }
}

File: java/src/main/java/com/experian/dvs/client/address/layout/LayoutLineVariable.java
package com.experian.dvs.client.address.layout;

import com.experian.dvs.client.address.layout.elements.AddressElement;
import com.experian.dvs.client.server.address.layout.RestApiAddressLayoutLine;

import java.util.List;
import java.util.Objects;

public class LayoutLineVariable implements LayoutLine{

    private String name;
    private Integer maxWidth;


    public LayoutLineVariable(String name) {
        this.name = name;
    }

    public LayoutLineVariable(RestApiAddressLayoutLine line) {
        this.name = Objects.toString(line.getLineName(), "");
        this.maxWidth = line.getMaxWidth();
    }

    @Override
    public String getName() {
        return this.name;
    }

    public void setName(String name) {
        this.name = name;
    }

    @Override
    public Integer getMaxWidth() {
        return this.maxWidth;
    }

    @Override
    public List<AddressElement> getElements() {
        return List.of();
    }

    public void setMaxWidth(Integer maxWidth) {
        this.maxWidth = maxWidth;
    }
}
